rtptransceiver.go

```diff
@@ -16,6 +17,10 @@ type RTPTransceiver struct {
 	receiver  atomic.Value // *RTPReceiver
 	direction atomic.Value // RTPTransceiverDirection
 
+	me     *MediaEngine
```

Comment:
Can you embed `api *API` instead? That is just the common pattern across `RTC*` objects we have right now.